coordinator/clients/prover-client/file-based-client/src/main/kotlin/net/consensys/zkevm/coordinator/clients/prover/GenericFileBasedProverClient.kt

```diff
@@ -118,10 +118,9 @@ open class GenericFileBasedProverClient<Request, Response, RequestDto, ResponseD
           FileMonitor.ErrorType.TIMED_OUT -> {
             SafeFuture.failedFuture<Path>(RuntimeException("Timeout waiting for response file=$responseFilePath"))
           }
-
-          else -> {
-            SafeFuture.failedFuture(RuntimeException("Unexpected error=$it"))
-          }
+          // else -> {
```

Comment:
This will trigger compilation error as `FileMonitor.ErrorType.TIMED_OUT` is currently the only possible value, comment out for now.